src/Cli/dotnet/SlnFileFactory.cs

```diff
@@ -57,7 +63,7 @@ private static SlnFile FromDirectory(string solutionDirectory)
                     solutionDirectory);
             }
 
-            FileInfo[] files = dir.GetFiles("*.sln");
+            FileInfo[] files = [..dir.GetFiles("*.sln"), ..dir.GetFiles("*.slnx")];
```

Comment:
Look for slnf?

Reply:
Same here, I think for most of the cases, we won't necessarily want .slnf files, but I think we could add a parameter to these methods to be reused more.
I think a good approach might be modifying this a bit more to be used more generically to look for sln(x/f) files